eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -30,14 +29,23 @@ steps:
       Write-Host "This script is not executing on Windows, skipping registry modification."
     }
   displayName: Enable Long Paths if Necessary
-  
-- pwsh: |
-    git clone https://github.com/${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }} ${{ parameters.WorkingDirectory }}/repo
-  displayName: Clone Documentation Repository
-  ignoreLASTEXITCODE: false
+
+- template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+  parameters:
+    SkipDefaultCheckout: true
+    Repositories:
+      - Name: ${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }}
+        Commitish: master
+        WorkingDirectory: ${{ parameters.WorkingDirectory }}/repo
+    Paths:
+      - docs-ref-services  # java, python, js
```

Comment:
Can we pass these in from the specific languages instead of hard-coding them here?

Reply:
Yep, will make that change